views/cellMove/cellMovePrisonerSearch.njk

```diff
@@ -0,0 +1,131 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "../macros/alertFlags.njk" import alertFlags %}
+{% from "../macros/categoryFlag.njk" import categoryFlag %}
+
+{% set title = 'Search for a prisoner' %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: '/'
+      },
+      {
+        text: "Change someone’s cell",
+        href: '/change-someones-cell'
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  })
+  }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+      }) 
+    }}
+  {% endif %}
+
+  <h1 class="govuk-heading-l">
+    {{title}}
+  </h1>
+
+  <form novalidate="novalidate" class="prisoner-search__form" data-test="prisoner-search-form">
+    <div class="horizontal-form govuk-!-margin-bottom-3">
+      {{ govukInput({
+        id: "keywords",
+        name: "keywords",
+        value: formValues.keywords,
+        label: {
+            text: "Enter a prisoner&#8217;s name or number"
+        },
+        classes: 'govuk-!-width-full',
+        attributes: {
+          'data-test': 'prisoner-search-keywords'
+        }
+      }) }}
+
+      {{ govukButton({
+        text: "Search",
+        type: "submit",
+        classes: "govuk-button--secondary"
+      }) }}
+    </div>
+  </form>
+
+  {% if hasSearched %}
+    <p class="govuk-body pull-right"><strong>Prisoners listed:</strong> {{totalOffenders}}</p>
+
+    {% if results.length %}
+      {% set rows = [] %}
+      {% for prisoner in results %}
+        {% set prisonerImage = '<img src="/app/images/' + prisoner.offenderNo + '/data" alt="Image of ' + prisoner.name + '" class="prisoner-search__results-list__image" />' %}
```

Comment:
This doesn't look right. Are there other places we do this?

Reply:
yeah - prisonerSearch.njk